examples/Infrastructure/InMemoryEmailGuard.php

```diff
@@ -20,7 +21,7 @@ final class InMemoryEmailGuard implements UniqueEmailGuard
 
     public function isUnique(string $email): bool
```

Comment:
Shouldn't this return `Promise<bool>` now?

Reply:
yes